tests/models/test_match_spec.py

```diff
@@ -717,3 +717,60 @@ def test_parse_build_number_brackets(self):
         #     "build_number": '>=3',
         # }
 
+
+class MatchSpecMergeTests(TestCase):
+
+    def test_merge_single_name(self):
+        specs = (MatchSpec('exact'), MatchSpec('exact 1.2.3 1'), MatchSpec('exact >1.0,<2'))
```

Comment:
Can you also add
```python
specs = (MatchSpec('exact 1.2.3 1'), MatchSpec('exact 1.2.3 2'))
with pytest.raises(ValueError):
    MatchSpec.merge(specs)
```
so that we have an "expected failure" test?

Reply:
Done.  Plus a bunch more tests added.